tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/reader/ServerBaseReader.scala

```diff
@@ -190,3 +198,57 @@ class NebulaReader(override val session: SparkSession, nebulaConfig: ServerDataS
     extends ServerBaseReader(session, nebulaConfig.sentence) {
   override def read(): DataFrame = ???
 }
+
+/**
+  * HBaseReader extends [[ServerBaseReader]]
+  *
+  */
+class HBaseReader(override val session: SparkSession, hbaseConfig: HBaseSourceConfigEntry)
+    extends ServerBaseReader(session, null) {
+
+  private[this] val LOG = Logger.getLogger(this.getClass)
+
+  override def read(): DataFrame = {
+    val cf       = hbaseConfig.columnFamily
+    val scanConf = HBaseConfiguration.create()
+    scanConf.set("hbase.zookeeper.quorum", hbaseConfig.host)
+    scanConf.set("hbase.zookeeper.property.clientPort", hbaseConfig.port)
+    scanConf.set(TableInputFormat.INPUT_TABLE, hbaseConfig.table)
+    hbaseConfig.fields.filter(field => !field.equalsIgnoreCase("rowkey"))
+    scanConf.set(TableInputFormat.SCAN_COLUMNS,
+                 hbaseConfig.fields
+                   .filter(field => !field.equalsIgnoreCase("rowkey"))
+                   .map(field => s"$cf:$field")
+                   .mkString(" "))
+    val fields = hbaseConfig.fields
+
+    val hbaseRDD: RDD[(ImmutableBytesWritable, Result)] = session.sparkContext.newAPIHadoopRDD(
+      scanConf,
+      classOf[TableInputFormat],
+      classOf[ImmutableBytesWritable],
+      classOf[Result])
+
+    val sourceSchema = hbaseConfig.sourceFieldSchema
+
+    val values = hbaseRDD.map(row => {
+      val values: ListBuffer[Any] = new ListBuffer[Any]
```

Comment:
`values` have shadowed maybe make reader confuse

Reply:
changed the first `values` to `rowRDD`